webapp/dev_settings.py

```diff
@@ -1,5 +1,4 @@
-from .settings import *
```

Comment:
Is this better? It's surely less robust against the app being renamed, so if you do rename this django app you have to remember to change this or things will break. I'd have thought a relative name would be better.